### PR TITLE
Create and update to-do using Express.js

### PR DESCRIPTION
**Pull request by avishekjana**

### CODE REVIEW

File: backend-dev-with-express/create-and-update-todo/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+As you already know, Sequelize is an ORM (Object Relational Mapping), and it creates an object-oriented layer between Node.Js and the PostgreSQL database. It helps us to work with data without even worrying about the raw SQL queries. 
+
+> Action: Open VS Code
+
+So let's define the **Todo** model:
+```js
+const Todo = sequelize.define(
+  'todos', 
+  { description: Sequelize.TEXT, isComplete: { type: Sequelize.BOOLEAN, defaultValue: false }, dueDate: Sequelize.DATE }
```
Comment (mjzac) [inline backend-dev-with-express/create-and-update-todo/README.md:19]:
```suggestion
  { description: Sequelize.TEXT, isComplete: { type: Sequelize.BOOLEAN, defaultValue: false }, dueDate: Sequelize.DATEONLY } 
```

File: backend-dev-with-express/create-and-update-todo/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+### Step 1: Define the TO-DO model using Sequelize
+Before we can write login inside our TO-DO endpoints, we have to define the **TODO** *model* using Sequelize.
+
+As you already know, Sequelize is an ORM (Object Relational Mapping), and it creates an object-oriented layer between Node.Js and the PostgreSQL database. It helps us to work with data without even worrying about the raw SQL queries. 
+
+> Action: Open VS Code
+
+So let's define the **Todo** model:
+```js
+const Todo = sequelize.define(
```
Comment (mjzac) [inline backend-dev-with-express/create-and-update-todo/README.md:17]: `sequelize` and `Sequelize` are used here. It is better to include the `require` statements then.

**Review by mjzac — CHANGES_REQUESTED** [review]

File: backend-dev-with-express/create-and-update-todo/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+We can test it out with a simple `curl` request from our terminal:
+````
+curl -d '{"description":"Go the gym","dueDate":"2022-07-02"}' -H "Content-Type: application/json" -X POST http://localhost:3000/todos
+````
+
+This `curl` call will create a new **todo** entry in our database, and will return the new database object to us.
+
+
+### Step 4: Complete the `update Todo` endpoint implementation
```
Comment (mjzac) [inline backend-dev-with-express/create-and-update-todo/README.md:64]: @avishekjana  The endpoint should be to mark a todo as complete, and not to update the description.

**Review by mjzac — CHANGES_REQUESTED** [review]

File: backend-dev-with-express/create-and-update-todo/README.md (hunk 3 above)
Comment (avishekjana) [reply backend-dev-with-express/create-and-update-todo/README.md:64]: OK

File: backend-dev-with-express/create-and-update-todo/README.md (hunk 2 above)
Comment (avishekjana) [reply backend-dev-with-express/create-and-update-todo/README.md:17]: Done

Comment (avishekjana) [reply backend-dev-with-express/create-and-update-todo/README.md]: It's no more required, as I've used model generated by sequelize-cli.

**Comment by avishekjana** [comment]
@mjzac Please review the script once again.

**Review by mjzac — CHANGES_REQUESTED** [review]

**Comment by avishekjana** [comment]
Resolved all comments/feedbacks, closing this issue.

**Pull request merged**